cmd/kyma/undeploy/cmd.go

```diff
@@ -297,6 +284,66 @@ func (cmd *command) deleteKymaNamespaces() error {
 	}
 }
 
+func (cmd *command) waitForNamespaces() error {
+
+	cmd.NewStep("Waiting for namespace termination")
```

Comment:
why is it called "termination" here? Is this something different from the "deletion" mentioned in line 272 or 296? 
If it's the same, let's say "deletion" here too.
```suggestion
	cmd.NewStep("Waiting for namespace deletion")
```